raster-foundry/raster-foundry
# Seed licenses data to the licenses table

## Overview

This PR creates a migration to seed [SPDX licenses list](https://github.com/sindresorhus/spdx-license-list/blob/master/spdx.json) to the licenses table in the DB. 

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~

### Notes

The values being inserted were transformed programmatically from the original json form to the string presented in this PR using [scripts here](https://github.com/aaronxsu/licenses-json-to-sql), which may be reused in similar cases.

## Testing Instructions

 * `mg update` and `mg apply`
 * Check if licenses are correctly seeded in the licenses table in the DB.

Closes #2955
